ajax/libs/jquery-xmlrpc/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery-xmlrpc",
+  "filename": "jquery.xmlrpc.min.js",
+  "version": "0.4.3",
+  "description": "A jQuery library for talking to XML-RPC services from JavaScript",
```

Comment:
@hare1039 Why not use the info from upstream? `description`, `keywords` and `author`. 
ref: https://github.com/timheap/jquery-xmlrpc/blob/v0.4.3/xmlrpc.jquery.json